### ISSUE TITLE
A11y_VisualStudioCodeClient_ Sidebar Bar Options_ Accessing The Git side bar options_Name Role Value: Read our docs is a link but Voice over is announcing as button

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#29400]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.
--- GitHubTags:
-A11yWCAG2.1; -A11yTCS; -A11ySev2; -Visual Studio Code Client; -BM-VisualStudioCodeClient-MAC-Aug2023; -Voiceover; -MAC; -Name Role Value; -WCAG4.1.2.
--- Environment and OS details:
Visual Studio Code insider for MAC version 1.82.0
Mac Version: 13.5
Screen reader: Voice Over
--- Repro Steps:
1.	Open Visual Studio Code Insiders editor
2.	Using "Ctrl+Shift+G" key navigate to the search side view bar option.
3.	Using "Tab" key navigate to Git side bar options.
4.	Turn on Voice over and Observe that Read our docs is a link but Voice over is announcing as button.
--- Actual Result:
Read our docs is a link but Voice over is announcing as button.
--- Expected Result:
Read our docs is a link and Voice over should be announcing it as Link.
--- User Impact:
Screen reader users may get confused if role of link is incorrectly defined as a button.
--- Attachment:

https://github-com/microsoft/vscode/assets/98728920/3af350cc-2af9-4c9c-87fd-057d1e56f2a7
